lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -909,15 +909,16 @@ def test_order_direction_query(self, http_query):
 
 
 @ddt.ddt
+@mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
 class GetCommentListTest(CommentsServiceMockMixin, SharedModuleStoreTestCase):
     """Test for get_comment_list"""
 
     @classmethod
-    @mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
     def setUpClass(cls):
         super(GetCommentListTest, cls).setUpClass()
         cls.course = CourseFactory.create()
 
+    @mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
```

Comment:
Patch defined on class has scope of class methods and all test methods; hence for setup, it was required to define it again.